--- include/netcdf.h.old	2018-07-05 14:45:59.596684316 -0400
+++ include/netcdf.h	2018-07-05 14:47:09.085917968 -0400
@@ -259,11 +259,11 @@
       are *not* enforced after version 4.5.0
 */
 /**@{*/
-#define NC_MAX_DIMS     1024 /* not enforced after 4.5.0 */
-#define NC_MAX_ATTRS    8192 /* not enforced after 4.5.0 */
-#define NC_MAX_VARS     8192 /* not enforced after 4.5.0 */
-#define NC_MAX_NAME     256
-#define NC_MAX_VAR_DIMS 1024 /**< max per variable dimensions */
+#define NC_MAX_DIMS    65536    /* max dimensions per file */
+#define NC_MAX_ATTRS    8192
+#define NC_MAX_VARS   524288    /* max variables per file */
+#define NC_MAX_NAME   256
+#define NC_MAX_VAR_DIMS      8     /* max per variable dimensions */
 /**@}*/

 /** This is the max size of an SD dataset name in HDF4 (from HDF4 documentation).*/
